src/NuGetPackageVerifier/CompositeRules/PackageDependenciesCompositeRule.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace NuGetPackageVerifier.Rules
+{
+    public class PackageDependenciesCompositeRule : IPackageVerifierRule
```

Comment:
Yes... of course. But if we run it on "timestamp" packages then our packages are always pre-release because they are `x.y.z-rtm-12345` so this rule will never check anything.